modules/travel_pay/spec/controllers/claims_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TravelPay::ClaimsController, type: :request do
+  let(:user) { build(:user) }
+
+  before do
+    veis_response = double
+    allow(veis_response).to receive(:body).and_return( {'access_token' => 'sample_token'} )
+    allow_any_instance_of(TravelPay::Client).to receive(:request_veis_token).and_return(veis_response)
+    btsss_ping_response = double
+    allow(btsss_ping_response).to receive(:status).and_return('online')
```

Comment:
`btsss_ping_response` is a `Faraday::Response`, and so I think it should be:

```ruby
allow(btsss_ping_response).to receive(:status).and_return(200)
```

which keeps the test double closer to the actual object.